cSpell.json

```diff
@@ -0,0 +1,289 @@
+        "PRDS",
+        "Premortem",
+        "PSLF",
+        "pushback",
+        "QASP",
+        "Qualtrics",
+        "Reengineering",
+        "reqs",
+        "résumé",
+        "résumés",
+        "roadmap",
+        "roadmapping",
+        "roadmaps",
+        "rotaters",
+        "roundtable",
+        "routable",
+        "screenshare",
+        "SDLC",
+        "Servicedesk",
+        "servicemember",
+        "SFMTA",
+        "signup",
+        "skillset",
+        "skillsets",
+        "Skillsoft",
+        "Slackbot",
+        "SLDP",
+        "Slidedeck",
+        "Smartsheet",
+        "Snyk",
+        "sportcoats",
+        "standups",
+        "Statuspage",
+        "strategize",
+        "submittals",
+        "telework",
+        "timeboxed",
+        "timeframe",
+        "timeframes",
+        "timesheet",
+        "timesheets",
+        "Tock",
+        "Tocked",
+        "Tocking",
+        "Tokencode",
+        "Touchpoints",
+        "trackpad",
+        "transformative",
+        "unaccrued",
+        "Unarchiving",
+        "underserved",
+        "Uptrends",
+        "USDS",
+        "USGSA",
+        "USWDS",
+        "Ventra",
+        "videoconference",
+        "Visio",
+        "VLTP",
+        "vuln",
+        "vulns",
+        "walkthrough",
+        "WCAG",
+        "Webex",
+        "Whiteboarding",
+        "wireframe",
+        "wireframes",
+        "womxn",
+        "workshopping",
+        "XKCD"
+    ],
+    "ignoreWords": [],
+    "patterns": [
+        {
+            "name": "Markdown links",
+            "pattern": "\\((.*)\\)",
+            "description": ""
+        },
+        {
+            "name": "Markdown code blocks",
+            "pattern": "/^(\\s*`{3,}).*[\\s\\S]*?^\\1/gmx",
+            "description": "Taken from the cSpell example at https://cspell.org/configuration/patterns/#verbose-regular-expressions"
+        },
+        {
+            "name": "Inline code blocks",
+            "pattern": "\\`([^\\`\\r\\n]+?)\\`",
+            "description": "https://stackoverflow.com/questions/41274241/how-to-capture-inline-markdown-code-but-not-a-markdown-code-fence-with-regex"
+        },
+        {
+            "name": "Link contents",
+            "pattern": "\\<a(.*)\\>",
+            "description": ""
+        },
+        {
+            "name": "Snippet references",
+            "pattern": "-- snippet:(.*)",
+            "description": ""
+        },
+        {
+            "name": "Snippet references 2",
+            "pattern": "\\<\\[sample:(.*)",
+            "description": "another kind of snippet reference"
+        },
+        {
+            "name": "Multi-line code blocks",
+            "pattern": "/^\\s*```[\\s\\S]*?^\\s*```/gm"
+        },
+        {
+            "name": "HTML Tags",
+            "pattern": "<[^>]*>",
+            "description": "Reference: https://stackoverflow.com/questions/11229831/regular-expression-to-remove-html-tags-from-a-string"
+        },
```

Comment:
I've been choosing to keep these simpler / more explicit but I think as long as the spellcheck continues to pass, if someone wants to adjust them, that sounds great to me. I'd love someone to ping me when they do too, because I'll be feeding all this info back into a central place to help more people supply this help to more repos.